Frontend/app/user/registration/registration.component.ts

```diff
@@ -38,10 +39,8 @@ export class RegistrationComponent {
     let data = err.json();
     this.submitInProgress = false;
     this.newModel();
-    let alertMessage = "Registation Failed:\n"
     for (let error of data['errors']) {
-      alertMessage += "Title:" + error['title'] + ", Message:" + error['message'] + "\n";
+      this.toastr.error(error['message'], error['title']);
```

Comment:
I know I keep bringing this up, but is a way for `error `to not have a message, title key and how will it be handled if not?